jfix-stdlib-id-generator/src/main/kotlin/ru/fix/stdlib/id/generator/AtomicIdGenerator.kt

```diff
@@ -100,5 +100,16 @@ class AtomicIdGenerator(
             }
         }
     }
+
+    fun extractTime(id: Long): Long {
```

Comment:
maybe rename methods to 
```
extractTimePart
extractServerIdPart
extractCounterPart
```
this way we get more clear meaning and also logically group this three methods  by suffix